core/psh/help-empty.c

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
Is there any possibility, we don't want to support `help` command?
One could argue, we should do it like that for consistency reasons, but I feel it's redundant, given each "virtual" command requires some additional code or even another file.